tests/unit/components/initials-avatar-test.js

```diff
@@ -11,20 +11,32 @@ test('shows first and last name initials', function(assert) {
   var component = this.subject({
     firstName: "Barack",
     lastName: "Obama",
-    company: "United States"
+    company: "United States",
+    email: "Obama@whitehouse.gov"
   });
   assert.equal(component.get("initials"), "BO");
 });
 
-test('shows company if first and last name not provided', function(assert) {
+test('shows company if email, first and last name not provided', function(assert) {
```

Comment:
I could probably add _another_ test that makes sure company has higher priority over email